graphql/student/mutations.ts

```diff
@@ -160,7 +202,73 @@ export async function updateStudent(context: GraphQLContext, student: Student, u
     await updateSessionUser(context, userForStudent(res));
 
     log.info(`Student(${student.id}) updated their account with ${JSON.stringify(update)}`);
+    return res;
+}
+
+async function studentRegisterPlus(data: StudentRegisterPlusInput, ctx: GraphQLContext): Promise<{ success: boolean; reason: string }> {
+    const log = logInContext('Student', ctx);
+    const { email, register, activate, screen } = data;
+    const screener = await getSessionScreener(ctx);
+
+    try {
+        if (register && register.email !== email) {
```

Comment:
/nit
We should try to be consistent in the way we are doing checks. Here we are checking `if (register)` whereas later it's `if(!!register)`.